lib/magic_words/version.rb

```diff
@@ -1,3 +1,3 @@
 module MagicWords
-  VERSION = "0.0.1"
+  VERSION = "0.0.2"
```

Comment:
please do not tick versions in PRs, we'll tick a version when we're ready for a new release

Reply:
Fixed